src/commands/moderation/clearwarns.ts

```diff
@@ -0,0 +1,61 @@
+import { CommandCategories, CommandDefinition, createErrorEmbed } from '../index';
+import { createEmbed } from '../../lib/embed';
+import Warn from '../../schemas/warn';
+
+export const clearwarns: CommandDefinition = {
+    names: ['clearwarns'],
+    description: 'Clears the specified number of warns a user has. `Arguments: <id> <count>`',
+    category: CommandCategories.MODERATION,
+    permissions: ['KickMembers'],
+    execute: async (message, args) => {
+        const invalidEmbed = createErrorEmbed('Please provide a valid user/id');
+
+        let id = args[0];
+        if (!id) {
+            await message.channel.send({ embeds: [invalidEmbed] }).catch(console.error);
+            return;
+        }
+
+        // in case of a mention
```

Comment:
for future additions, we should create a common mention/id checker if the lib doesnt have one (back in v11 it did have something to make it easy, idk now)